tests/pytorch/test_fused_attn.py

```diff
@@ -724,6 +724,10 @@ def _run_transformer_layer(
     # Create input tensor
     inp = torch.randn(config.max_seqlen_q, config.batch_size, config.hidden_size,
             dtype=dtype, device="cuda", requires_grad = True)
+    # In case the format to be tested is batch-first, need to transpose the
+    # input tensor.
+    if qkv_layout == "bshd":
```

Comment:
Maybe change `qkv_layout` to `qkv_format` here and on L708, since we've been using `qkv_format` in `test_transformer_layer` and `qkv_layout` actually means the more detailed layouts (like `sbhd_sbhd_sbhd`). Thanks!